src/openrct2/object/Object.cpp

```diff
@@ -329,6 +329,49 @@ std::unique_ptr<IStream> ObjectAsset::GetStream() const
     return {};
 }
 
+/*
+std::string VersionString(Version version)
```

Comment:
If this turns out to work, this will need to be removed before it’s merged. (And put into a PR of its own.)